### PR TITLE
Fixes to custom resolver docs

### PR DESCRIPTION
**Pull request by nathforge**
* Changed super class call.
* Updated config example.

### CODE REVIEW

**Comment by jamesroutley** [comment]
Hey @nathforge thanks for the PR! I've added a comment but it looks good to me apart from that.

James

File: docs/advanced_patterns.rst [hunk 1]
```diff
@@ -83,11 +83,10 @@ This resolver can be used in a stack config file with the following syntax::
 
   template_path: <...>
   parameters:
-    param1:
-      <your resolver name>: <value>  # <your resolver name>
-                                     # should match Hook.name.
-                                     # <value> will be passed to the
-                                     # resolver's resolve() method.
+    param1: !<your_resolver_name>: <value> # <your resolver name> is the lower camel-case version
```
Comment (theseanything) [inline docs/advanced_patterns.rst]: this needs to be `!<your_resolver_name> <value>` - without the colon between resolver name and value

**Comment by theseanything** [comment]
@nathforge thanks! Just a small error in the syntax, but good spot!

**Comment by nathforge** [comment]
Thanks both, have updated!

**Pull request merged**